superset/db_engine_specs/hive.py

```diff
@@ -179,11 +193,15 @@ def convert_to_hive_type(col_type: str) -> str:
             bucket_path,
             os.path.join(upload_prefix, table_name, os.path.basename(filename)),
         )
+        engine = cls.get_engine(database)
+
+        if if_exists == "replace":
+            engine.execute(f"DROP TABLE IF EXISTS {full_table_name}")
```

Comment:
this looks ripe for sql injection, but honestly, the existing lines below are as well. This is blocked on enabling the csv upload feature to a datasource, so that security might be ok for now?

Reply:
There's a few ways around this, but for now I think we can leave these as-is. Perhaps add a TODO here so we can sweep them all up in one go later?

Reply:
Yeah, this is scary. Let's see if we can explore some of the possible solutions for this to get around adding another instance of SQL injection vulnerability.

Reply:
Looks like this might not be possible as per https://stackoverflow.com/a/43879809?